libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1842,7 +1842,8 @@ void WaveTrack::HandleClear(
       InsertClip(std::move(clip)); // transfer ownership
 
    if (!moveClipsLeft && shiftClipAtT1ToT0)
-      if (const auto clip = GetClipAtTime(t1))
+      if (const auto clip = GetClipAtTime(t1);
+          clip && clip->GetPlayStartTime() == t1)
```

Comment:
I see an exact comparison of floating point values, and I want to know whether they were first rounded to a multiple of the same inverted rate.  That's not obvious in context here.

Reply:
And:  please amend the comment for this commit, with some line breaks

Reply:
> I see an exact comparison of floating point values, and I want to know whether they were first rounded to a multiple of the same inverted rate. That's not obvious in context here.

You're right, adding some rounding at the beginning of that method.

Reply:
> And: please amend the comment for this commit, with some line breaks

Where do you see the need for line breaks?